src/fstests/fstests.jinja2

```diff
@@ -36,6 +36,16 @@ class Job(BaseJob):
         except Exception as e:
             print('Kernel build error:', e)
 
+    def _run_smoke_tests(self):
+        try:
+            result = subprocess.run("kvm-xfstests smoke", check=True, shell=True)
```

Comment:
In general it's better to pass command and arguments as a sequence if you pass it as a string it's interpretation becomes platform dependant.
```suggestion
            result = subprocess.run(["kvm-xfstests", "smoke"], check=True, shell=True)
```

Reply:
Do we need to run the command in the shell (shell=True)?

Reply:
no, we dont. We just need to build the kernel...